### PR TITLE
Update UG with Add Lesson and Delete Lesson for Student 

### PR DESCRIPTION
**Pull request by eeliana**
Fix #63

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -142,6 +142,52 @@ Examples:
...
+
+**Types of lesson:**
+* Recurring
+* Makeup
+
+**Lesson fields:**
+* Date of lesson
+* dd MMM yyyy e.g. Tue, 02 Jan 2018
+* Start time HH:mm
+* End time HH:mm
```
Comment (lingshanng) [inline docs/UserGuide.md:157]:
```suggestion
* Date of lesson `dd MMM yyyy`
  * e.g. `Tue, 02 Jan 2018`
* Start time `HH:mm`
* End time `HH:mm`
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -142,6 +142,52 @@ Examples:
...
+* Subject
+* Homework
+
+Format: `ladd INDEX [recurring/] date/<dd MMM yyyy> start/<HH:mm> end/<HH:mm> subject/SUBJECT [hw/HOMEWORK]`
+
+<div markdown="span" class="alert alert-primary">
+:bulb: The type of lesson will be inferred from the presence of the -recurring flag.
```
Comment (lingshanng) [inline docs/UserGuide.md:164]:
```suggestion
:bulb: The type of lesson will be inferred from the presence of the `recurring/` flag.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -142,6 +142,52 @@ Examples:
...
+Format: `ldelete INDEX LESSON_INDEX`
+
+* Deletes the lesson of specified `LESSON_INDEX` for the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The lesson index refers to the index number shown in the lesson list of the person. 
+* The index and lesson index **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+* `list` followed by `ldelete 2 1` deletes the 1st lesson for the 2nd person in the address book.
+* `find Betsy` followed by `delete 1 1` deletes the 1st lesson for the 1st person in the results
```
Comment (lingshanng) [inline docs/UserGuide.md:188]:
```suggestion
* `find Betsy` followed by `ldelete 1 1` deletes the 1st lesson for the 1st person in the results 
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -142,6 +142,52 @@ Examples:
...
+
+Format: `ladd INDEX [recurring/] date/<dd MMM yyyy> start/<HH:mm> end/<HH:mm> subject/SUBJECT [hw/HOMEWORK]`
+
+<div markdown="span" class="alert alert-primary">
+:bulb: The type of lesson will be inferred from the presence of the -recurring flag.
+</div>
+
+Examples:
+
+* `list` followed by `ladd 1 recurring date/16 Sep 2021 start/15:00 end/16:00 subject/Math` (recurring lesson)
```
Comment (lingshanng) [inline docs/UserGuide.md:169]:
```suggestion
* `list` followed by `ladd 1 recurring/ date/16 Sep 2021 start/15:00 end/16:00 subject/Math` (recurring lesson)
```

File: docs/UserGuide.md (hunk 1 above)
Comment (lingshanng) [reply docs/UserGuide.md:157]: is this format better?

**Review by lingshanng — COMMENTED** [review]
I only review UG

File: docs/UserGuide.md (hunk 2 above)
Comment (eeliana) [reply docs/UserGuide.md]: It seems like backticks does not work in the div. Will replacing them with double quotes be better?

**Review by lingshanng — APPROVED** [review]
LGTM

**Pull request merged**